src/components/inline-loading/inline-loading.hbs

```diff
@@ -11,9 +11,9 @@
     <div data-inline-loading-spinner class="{{@root.prefix}}--loading {{@root.prefix}}--loading--small">
       <svg class="{{@root.prefix}}--loading__svg" viewBox="-75 -75 150 150">
         {{#if componentsX}}
-        <circle class="{{@root.prefix}}--loading__background" cx="0" cy="0" r="37.5" />
+        <circle class="{{@root.prefix}}--loading__background" cx="0" cy="0" r="30.5" />
         {{/if}}
-        <circle {{#if componentsX}}class="{{@root.prefix}}--loading__stroke"{{/if}} cx="0" cy="0" r="37.5" />
+        <circle {{#if componentsX}}class="{{@root.prefix}}--loading__stroke"{{/if}} cx="0" cy="0" r="30" />
```

Comment:
Seems that the `r` attribute here is applied to `v9` as well, and thus the change here need to be roped off.